yarn-project/noir-contracts/src/contracts/noir-aztec3/src/oracle/debug_log.nr

```diff
@@ -1,9 +1,25 @@
-
 // Utility function to console.log data in the acir simulator
 
 #[oracle(debugLog)]
-fn debug_log_oracle(_msg: Field) -> Field {}
+fn debug_log_oracle<T, N>(_msg: T, _num_args: Field) -> Field {}
```

Comment:
wait wut i had no idea you could do this with generics! / have multiple implementations of `debugLog` thats nice!